.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
 
 before_script:
     # Get the test cases
-    - git clone -b develop https://github.com/su2code/TestCases.git ./TestData
+    - git clone -b feature_bugfix_DT1st_DiscAdjFluid https://github.com/su2code/TestCases.git ./TestData
```

Comment:
You can do a pull request to the develop branch of the TestCases first to incorporate the new case.